config/web.xml

```diff
@@ -70,6 +70,15 @@
         <url-pattern>/admin/status</url-pattern>
     </servlet-mapping>
 
+    <servlet>
+        <servlet-class>edu.caltech.ipac.firefly.server.servlets.HealthCheck</servlet-class>
+        <servlet-name>healthz</servlet-name>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>healthz</servlet-name>
+        <url-pattern>/healthz</url-pattern>
```

Comment:
why `healthz` and not just `health`?

Reply:
> It historically comes from Google’s internal practices. They're called "z-pages".

I am following Kubernetes's convention which is from Google.  It is also popular in the kubernetes/docker community.
The 'z' is also to avoid any conflict with services endpoint.